docs/src/content/docs/ru/guides/i18n.mdx

```diff
@@ -55,23 +62,23 @@ Starlight предоставляет встроенную поддержку д
        - docs/
          - ar/
          - en/
-         - zh-cn/
+         - ru/
 
    </FileTree>
 
-3. Теперь вы можете добавлять контент в ваши языковые каталоги. Используйте одно и то же имя файла, чтобы связать страницы на разных языках и использовать все функции i18n Starlight, включая резервной контент, уведомления о переводах и многое другое.
+3. Теперь вы можете добавлять файлы содержимого в свои языковые каталоги. Используйте одно и то же имя файла для связывания страниц на разных языках и воспользуйтесь полным набором функций i18n Starlight, включая резервный контент, уведомления о переводе и многое другое.
```

Comment:
I'm not sure that 'файлы содержимого' is clear to everyone. I think it's better to stick with 'контент' especially since you used the word 'контент' before and continue to use it

Reply:
You are right, thanks for the remark.